scripts/HPC_chatbot_preprocessor/parsed_mds/generic/account/account_paragraph_1.txt

```diff
@@ -0,0 +1,36 @@
+Getting an HPC Account
+Getting ready to request an account
+All users of AUGent can request
+an
+account on the HPC, which is part of the Flemish Supercomputing Centre (VSC).
+See HPC policies for more information on who is entitled to an account.
+The VSC, abbreviation of Flemish Supercomputer Centre, is a virtual
```

Comment:
is this explained anywehre else? maybe add this the acronym is actually form dutch etc